### DISCUSSION TITLE
Option to switch between local and global score ranking in result screen

### DISCUSSION THREAD

**Discussion by BurningFlames2314** [ppy/osu#18467] [Ideas] [answered]
In the song select screen, I like to keep my tab on the details tab instead of the scores, so it is slightly inconvenient whenever I want to see how well I did compared to all my other attempt, switching back and forth between the 2 tabs. So I was thinking maybe there could be an option where it would show my score compared to the top 50, where I never am near the top 50 so it doesnt really help me :/ It could show instead how people did with the local scores. Im pretty sure this is probably an idea that already has been stated but i could find any discussion on it so... ¯\_(ツ)_/¯

**Comment by integerrr** [comment, marked as answer]
Song select is getting a WIP redesign which should make it so that we don't have to screw around with what ranking is shown in the result screen by keeping the beatmap stats up all the time
https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=1%3A1276

**Reply by ikkue** [reply]
I don't know if this answer addresses the result screen after you've finished a map. Right now, even if the leaderboards on the Song Select screen is set to the Local Leaderboards, the result screen after you've finished the map still only displays the Global Leaderboards.

**Reply by BurningFlames2314** [reply]
Well, i juat kinda assume from the way the reply was phrased that it is unlikely it will be implemented at least for now (maybe in the future)... probably because its not very high on the list in terms of priority, this suggestion is more of a quality of life update i suppose
